### PR TITLE
Seth branch

### PR DESCRIPTION
**Pull request by DarthSethious1**
Just added a few comments to help with understanding

### CODE REVIEW

File: Buffer.h [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #pragma once
 
 #ifndef BUFFER_SIZE
-#define BUFFER_SIZE 200
```
Comment (bbrk24) [inline Buffer.h:4]: This is still here

**Review by bbrk24 — CHANGES_REQUESTED** [review]
The comments themselves look good, but since you reused the same branch as our example PR on Tuesday, the changes from that are also present. In general, I make a new branch for each PR, with very few exceptions.

**Comment by DarthSethious1** [comment]
So should i create a new branch everytime I start a new PR?

File: Buffer.h (hunk 1 above)
Comment (DarthSethious1) [reply Buffer.h:4]: So I would start a new branch every time I do a new PR? Or expect to do a new PR?

**Comment by bbrk24** [comment]
> So should i create a new branch everytime I start a new PR?

Correct, that's what I generally do. Don't worry about closing this one and creating a new PR with the same changes on another branch, though. In this case it's easy enough to add another commit that changes the 100 back to 200.

**Comment by bbrk24** [comment]
Superseded by #9

**Pull request closed without merging**